src/group_theory/coset.lean

```diff
@@ -479,6 +479,15 @@ have h : ∀ {x : α ⧸ s} {a : α}, x ∈ t → a ∈ s →
   left_inv := λ ⟨a, ha⟩, subtype.eq $ show _ * _ = a, by simp,
   right_inv := λ ⟨⟨a, ha⟩, ⟨x, hx⟩⟩, show (_, _) = _, by simp [h hx ha] }
 
+/-- If `K ≤ L`, then there is an embedding `K/(H ⊓ K) ↪ L/(H ⊓ L)` -/
+def le_quot_map (H : subgroup α) {K L : subgroup α} (h : K ≤ L) :
```

Comment:
I also PRed this (#10901), since it was used in the proof of `relindex_ne_zero_trans`.

Reply:
Ok great, I've removed this.